www/index.js

```diff
@@ -8,20 +8,16 @@ const GRID_COLOR = "#CCCCCC";
 const DEAD_COLOR = "#FFFFFF";
 const ALIVE_COLOR = "#000000";
 
+const canvas = document.getElementById("game-of-life-canvas");
+
 // Construct the universe, and get its width and height.
-const universe = Universe.new(64, 64);
+const universe = Universe.new(canvas.width, canvas.height);
 const width = universe.width();
 const height = universe.height();
 
 // Initialize it to some interesting state
 universe.initialize();
```

Comment:
Guess this should be called in `adjustLayout` after resizing, but I don't see the point as users don't resize their browser and can do a refresh if needed.

Reply:
Doing a refresh after resizing the browser doesn't seem to have the desired effect.

Reply:
You're right, was hard refreshing out of habit where it works ok. Will try to fix it.

Reply:
`window.onload` _should_ work per [documentation and example](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event). Had some limited success with `document.addEventListener("readystatechange", adjustLayout);` it works on Chrome only every nth time - seems like a race condition, but not at all in Firefox. Inspecting network didn't reveal much, there seems to be clues about the order of loading, but then the order of loading that works in Chrome doesn't work in Firefox. Configuring webpack to act like a "production" server didn't help, now trying to run serve it with nginx now to see if it makes a difference.

Reply:
Brute force approach seems to be the only way to get it working: 60b1161

Reply:
Disclaimers: performance isn't affected and `innerWidth` whose type hasn't been documented anywhere (or I just can't find it) when console.logged seems to be an integer, not a float so equality comparison seems safe (also tried logging its type, spits out number - as a js noob have no idea what that means).

Reply:
Seems to work, but then somehow it is all wrong. The actual size of the universe is determined from the size of the canvas, but this happens once and it happens at a point where the window contents haven't even been laid out properly. So what we are looking at is only a portion of the universe and that is most likely not what we want.

Maybe we first need to specify how we actually want this to behave. I could imagine two possible scenarios:
- Have fixed square dimensions for the universe, or maybe an option menu in the navbar that allows to pick from a set of predefined sizes. The canvas would then be drawn centered so that it fills as much of the available space as possible.
- Have a fixed cell size, or maybe an option menu in the navbar that allows to pick from a set of predefined cell sizes. The universe dimensions would then calculated so that it fills the available space. Resizing the window would cause the universe to be reallocated. That means the current game would end and a new game with new dimensions would start.

Reply:
Scenario 1 (cell size resizing) at 206b804e7fb240f9e07b0555406f25247e9441c6

Reply:
Works much better now. There's a problem though, if you resize the window with the universe paused, the cells aren't resized. I guess we need to add back the "resize" event handler.

Reply:
I've pushed another commit implementing the suggested changes and also doing the cleanups you asked for.

Reply:
Thanks. If you're happy, feel free to merge.